src/pages/Application/index.js

```diff
@@ -1,6 +1,5 @@
 import React, {useState, useCallback} from 'react';
-import Button from '../components/Button';
-import Dialog from '../components/ِDialog';
+import {Button, Dialog, TextInput} from '../../components';
```

Comment:
```suggestion
import {Button, Dialog} from '../../components';
```
Please avoid unused variables